packages/back-end/types/sdk-connection.d.ts

```diff
@@ -8,6 +8,7 @@ export interface ProxyConnection {
   version: string;
   error: string;
   lastError: Date | null;
+  sseEnabled?: boolean;
```

Comment:
The whole point of this is to enable sse for people not using a proxy server. So I would move this field to the top level of Sdk connections, not under proxy.

Reply:
my bad, I do recall that we were going to proxy behind the scenes on cloud, right? not explicitly via a connection setting.